packages/app/rollup.config.js

```diff
@@ -2,22 +2,15 @@ import alias from '@rollup/plugin-alias';
 import { defineConfig } from 'rollup';
 import dts from 'rollup-plugin-dts';
 
+import { aliasEntries } from '../shared/rollup.utils.js';
 import { externals } from './vite.config.js';
 
 export default defineConfig({
   input: './dist-ts/index.d.ts',
   output: [{ file: 'dist/index.d.ts', format: 'es' }],
   external: [...externals, /\.css$/u],
   plugins: [
-    alias({
-      entries: [
-        {
-          // Make sure rollup-plugin-dts can find shared types
-          find: '@h5web/shared',
-          replacement: '../shared/dist-ts/index.d.ts',
-        },
-      ],
-    }),
+    alias({ entries: aliasEntries }), // make sure rollup-plugin-dts can find type declatations in @h5web/shared
```

Comment:
All the rollup config files need the same alias entries, so I've put the code in a shared file, `packages/shared/rollup.utils.js`.